uwblueprint/feeding-canadian-kids
# Get OnboardingRequests

## Zenhub ticket link

[Get Onboarding Request](https://app.zenhub.com/workspaces/fck-627480c10d8974001872b0ef/issues/uwblueprint/feeding-canadian-kids/12)



## Implementation description
* add implementations to `getAllOnboardingRequests` which can be filtered by status and role
* add implementations to `getOnboardingRequestById` 
* added resolvers ^ and tests as well



## Steps to test
1. Run `docker exec -it fck_backend /bin/bash -c "pip install -e . && pytest -vv"` and verify that no test_onboarding_request files are failing



## What should reviewers focus on?
* 


## Checklist
